main.md

```diff
@@ -679,16 +679,137 @@ TBD
 
 # Privacy Considerations {#privacy_considerations}
 
-## Claim Names
+Privacy protection can be considered from the following principles per ISO/IEC 29100.  
```

Comment:
Is there a good link we can provide here? I fell that having the full list makes sense only if it is not readily available somewhere else.